.github/workflows/ci.yml

```diff
@@ -145,6 +153,6 @@ jobs:
         run: ./phpunit --coverage-clover=coverage.xml
 
       - name: Send code coverage report to Codecov.io
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-        run: bash <(curl -s https://codecov.io/bash) || true
```

Comment:
Looks like this does not work in Windows, that's why I switched to using the action.

Reply:
You need to specify the bash shell on Windows because Powershell is the default, so:
```
        run: bash <(curl -s https://codecov.io/bash) || true
        shell: bash
```

Perhaps the action is better though, for future maintenance.